workflow/scripts/preprocess/extract_metadata_tile.py

```diff
@@ -1,5 +1,7 @@
 from lib.preprocess.preprocess import extract_metadata_tile
 
+print(snakemake.input)
```

Comment:
Is this here because you are trying to debug?

Reply:
Yep, good catch! Removed unnecessary print statements in https://github.com/cheeseman-lab/brieflow/pull/13/commits/583d15c08147212b3b1ddf40c9acbf1a76484f46